agent/lib/tasks/release.rake

```diff
@@ -11,7 +11,7 @@ namespace :release do
   end
 
   desc 'Build all'
-  task :build => [:build_docker, :build_ubuntu] do
+  task :build => [:build_docker, :build_ubuntu, :build_ubuntu_xenial] do
```

Comment:
Hi guys, sorry for asking, any forecast for release this fix or should I go back to the ubuntu trusty?
If you need some tests, i'm available ok